src/topp/CometAdapter.cpp

```diff
@@ -272,19 +310,24 @@ class TOPPCometAdapter :
 
     os << "num_threads = " << getIntOption_("threads") << "\n";                         // 0=poll CPU to set num threads; else specify num threads directly (max 64)
 
+    os << "fragindex_min_ions_score = " << getIntOption_("fragindex_min_ions_score") << "\n";   // minimum number of matched fragment ion index peaks for scoring
+    os << "fragindex_min_ions_report = " << getIntOption_("fragindex_min_ions_report") << "\n"; // minimum number of matched fragment ion index peaks for reporting(>= fragindex_min_ions_score)
+    os << "fragindex_num_spectrumpeaks = " << getIntOption_("fragindex_num_spectrumpeaks") << "\n"; // number of peaks from spectrum to use for fragment ion index matching
+    os << "fragindex_min_fragmentmass = " << getDoubleOption_("fragindex_min_fragmentmass") << "\n";    // low mass cutoff for fragment ions
+    os << "fragindex_max_fragmentmass = " << getDoubleOption_("fragindex_max_fragmentmass") << "\n";    // high mass cutoff for fragment ions
+    os << "fragindex_skipreadprecursors = " << (int)(getStringOption_("fragindex_skipreadprecursors") == "true") << "\n";   // 0=read precursors to limit fragment ion index, 1=skip reading precursors
+
     // masses
     map<String,int> precursor_error_units;
     precursor_error_units["amu"] = 0;
     precursor_error_units["mmu"] = 1;
     precursor_error_units["ppm"] = 2;
 
     map<string,int> isotope_error;
-    isotope_error["off"] = 0;
-    isotope_error["0/1"] = 1;
-    isotope_error["0/1/2"] = 2;
-    isotope_error["0/1/2/3"] = 3;
-    isotope_error["-8/-4/0/4/8"] = 4;
-    isotope_error["-1/0/1/2/3"] = 5;
+for (size_t i = 0; i < isotope_error_str.size(); ++i) 
+{
+    isotope_error[isotope_error_str[i]] = i;
+}
```

Comment:
```suggestion
```

Reply:
remove `map<string,int> isotope_error;` as well. Not needed anymore

Reply:
removed